website/cue/reference/releases/0.40.1.cue

```diff
@@ -0,0 +1,35 @@
+package metadata
+
+releases: "0.40.1": {
+	date:     "2024-08-26"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is pleased to announce version 0.40.1!
+
+		This is the first release with packages signed with the new GPG key as part of [Datadog's key
+		rotation](https://docs.datadoghq.com/agent/guide/linux-key-rotation-2024/?tab=debianubuntu).
+		This should be a transparent change for users as the package repository setup script
+		[setup.vector.dev](https://setup.vector.dev) has already been importing the new key;
+		however, if you were manually managing the trusted GPG keys, you will need to update to
+		the [newer one](https://keys.datadoghq.com/DATADOG_APT_KEY_CURRENT.public).
```

Comment:
This would apply to both APT and RPM keys - and it's probably more obvious for the end users to get the specific keys - `DATADOG_APT_KEY_C0962C7D.public` and `DATADOG_RPM_KEY_B01082D3.public` so they can tell they have the right key without checking the expiration (vs the CURRENT alias).

Reply:
Aha, yes, I missed the `APT` in the name and thought it was just one key used for both. 👍